### DISCUSSION TITLE
How to use poincare math now?

### DISCUSSION THREAD

**Discussion by jin-zhe** [geoopt/geoopt#185] [Q&A] [answered]
Hi there! Please correct me if I'm wrong but it appears that `geoopt.manifolds.poincare.math` has been refactored into `geoopt.manifolds.stereographic.math`.

May I know how do I go about performing `geoopt.manifolds.poincare.math.lambda_x(x, *, c=1.0)` now?

Basically I'm not sure about the relationship between parameter `k` (sectional curvature) and the previous parameter `c` (call negative curvature).

**Comment by ferrine** [comment, marked as answer]
You should use negative k. Or I'd suggest to instantiate a Poincaré ball, that is a more readable way to do that

**Reply by jin-zhe** [reply]
Thanks! So I take it that the recommended way is to do something like the following?

```py
pb = geoopt.manifolds.PoincareBall(c=1.0)
pb.lambda_x(x)
```

**Comment by ferrine** [comment]
Right
